src/sas/sascalc/pr/invertor.py

```diff
@@ -583,13 +587,17 @@ def estimate_alpha(self, nfunc):
 
             # Perform inversion to find the largest alpha
             out, _ = pr.invert(nfunc)
+            #This invert is correct
+
             elapsed = time.time() - starttime
             initial_alpha = pr.alpha
             initial_peaks = pr.get_peaks(out)
+            #Inital Alpha correct, suggested not
 
             # Try the inversion with the estimated alpha
             pr.alpha = pr.suggested_alpha
             out, _ = pr.invert(nfunc)
+            #This invert different
```

Comment:
590, 595 and 600 are not useful comments.

Reply:
Removed, they were just there during debugging, forgot they were left.